ui/pages/confirmations/components/confirm/info/token-transfer/token-transfer.tsx

```diff
@@ -1,8 +1,26 @@
 import React from 'react';
+import { useSelector } from 'react-redux';
+import { selectConfirmationAdvancedDetailsOpen } from '../../../../selectors/preferences';
+import { AdvancedDetails } from '../shared/advanced-details/advanced-details';
+import { GasFeesSection } from '../shared/gas-fees-section/gas-fees-section';
 import SendHeading from '../shared/send-heading/send-heading';
+import { TokenDetailsSection } from './token-details-section';
+import { TransactionFlowSection } from './transaction-flow-section';
 
 const TokenTransferInfo = () => {
-  return <SendHeading />;
+  const showAdvancedDetails = useSelector(
+    selectConfirmationAdvancedDetailsOpen,
+  );
+
+  return (
+    <>
+      <SendHeading />
+      <TransactionFlowSection />
+      <TokenDetailsSection />
+      <GasFeesSection />
+      {showAdvancedDetails && <AdvancedDetails />}
```

Comment:
Minor, I wonder if it's worth encapsulating the selector of the preference within `AdvancedDetails` itself, we could add a property like `ignorePreference` if we wanted it always present in specific screens.